static/app/views/performance/transactionSummary/transactionSpans/spanSummary/spanSummaryHeader.tsx

```diff
@@ -10,11 +10,11 @@ import {formatMetricUsingUnit} from 'sentry/utils/metrics/formatters';
 import {DataTitles} from 'sentry/views/starfish/views/spans/types';
 
 type Props = {
-  avgDuration: number;
-  spanCount: number;
-  spanDescription: string;
-  spanOp: string;
-  timeSpent: number;
+  avgDuration: number | undefined;
+  spanCount: number | undefined;
+  spanDescription: string | undefined;
+  spanOp: string | undefined;
```

Comment:
Is there a case where `spanOp` might be `undefined` now? (The other ones are clear from the changes, just not sure about that one.)

Reply:
`spanOp` should always be returned AFAIK, but since the header isn't handling any loading state at the moment it could be `undefined` if the query has not finished. Speaking of which though, I should probably make use of the loading state in this case instead